StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/v1-elements-sessions/CustomPaymentMethod.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  CustomPaymentMethod.swift
+//  StripePaymentSheet
+//
+//  Created by Nick Porter on 3/5/25.
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+@_spi(STP) import StripePayments
+
+struct CustomPaymentMethod: Decodable {
```

Comment:
Noting that all these properties (besides type) are intentionally optional, see an example response here for why: https://github.com/stripe/stripe-ios/pull/4637/files#diff-c71d9097e968f7c5fd01a4cb9425c80f53998412b3026808eca160f1ca08b35fR139